browser/src/control/Control.JSDialog.js

```diff
@@ -600,10 +600,17 @@ L.Control.JSDialog = L.Control.extend({
 		for (var i = 0; i < dialogKeys.length; i++) {
 			var autoFilterDialogId = dialogKeys[i];
 			var dialog = this.dialogs[autoFilterDialogId];
+			var isAutoFillFromOnMouseUp = this.map._docLayer.isAutoFillFromOnMouseUp;
```

Comment:
I don't think we should have `isAutofilter == true` for that popup....
How we set this up? is that needed? maybe just don't use it for autofill?

Reply:
I think this is very useful since when sheet tabs are changed, it automatically closes the tabs (including mine - autoFillPopup) so we don't need to fire `close` for every situation that we need to close popup. 

This code block also runs after when we use `autoFill` with `AutoFill marker box`. Since I open the popup before the cells are filled with data, this for loop closes `autoFill` popup immediately after it opens, therefore we can't see the popup. 

That's why I want to prevent that `this.close(autoFilterDialogId, true)` call only if we use `autoFill` with `AutoFill marker box`. (which triggers with `onMouseUp` in `browser/src/canvas/sections/AutoFillMarkerSection.ts`)

Reply:
I agree with the effect, but I'm concerned about code, if it is not misleading :)

My question is: why    `isAutofilter` is set for auto fill popup which is not `autofilter` ?

Reply:
To be honest. I actually don't know why :) I think mentionPopup and formulaAutoComplete popup is also `true` for ` isAutofilter`.

Reply:
`instance.isAutofilter = instance.isDocumentAreaPopup && this.map._docLayer.isCalc();`  in Control.JSDialog.js

Could you put your check for `id` there? so it will NOT be `isAutofilter == true` ?

Reply:
Thanks, I have missed that part. 
Done with -> https://github.com/CollaboraOnline/online/pull/9135/commits/1bdaba888f3964d8c74bc2fc1572d3545109d7ea